lib/Sema/CSFix.h

```diff
@@ -136,6 +136,9 @@ enum class FixKind : uint8_t {
   /// If there is a matching inaccessible member - allow it as if there
   /// no access control.
   AllowInaccessibleMember,
+
+  /// Allow implicit coersion from optional to any
+  AllowImplicitCoercionToAny,
```

Comment:
I think it would make sense to rename this to `WarnImplicitCoercionToAny` or something like that because these are allowed already (same goes for the fix class).

Reply:
Done!